x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/entity_store/entity_definitions/entity_descriptions/common.ts

```diff
@@ -34,3 +37,25 @@ export const getCommonFieldDescriptions = (ecsField: BaseECSEntityField): FieldD
     }),
   ];
 };
+
+export const getEntityFieldsDescriptions = (rootField?: EntityType) => {
+  const prefix = rootField ? `${rootField}.entity` : 'entity';
+
+  return [
+    newestValue({ source: `${prefix}.name`, destination: 'entity.name' }),
+    newestValue({ source: `${prefix}.source`, destination: 'entity.source' }),
+    newestValue({ source: `${prefix}.type`, destination: 'entity.type' }),
+    newestValue({ source: `${prefix}.sub_type`, destination: 'entity.sub_type' }),
+    newestValue({ source: `${prefix}.url`, destination: 'entity.url' }),
+    newestValue({
```

Comment:
No need for `entity.behavior` ?

Reply:
We don't have any attribute under `entity.behavior` right now. I'm waiting on @MikePaquette